tests/resources/v3/manifest-00000004119.json

```diff
@@ -9712,7 +9712,7 @@
           "type" : "Range",
           "label" : {
             "de" : [
-              "87. Analisis of Beauty Plate I"
+              "87. Analysis of Beauty Plate I"
```

Comment:
Please undo this change. This is a sample manifest; the Range label reflects an existing typo in the original print.